util/dockerfiles/docker-bake.hcl

```diff
@@ -126,79 +206,40 @@ target "ubuntu-24-04_min-dependencies" {
   tags = ["${IMAGE_URI}/ubuntu-24.04_min-dependencies:${TAG}"]
 }
 
-target "gcc-compilers-base-20-04" {
-  name = "gcc-compilers-${replace(ver, ".", "-")}"
-  inherits = ["common"]
-  context = "ubuntu-20.04_gcc-version"
-  dockerfile = "Dockerfile"
-  matrix = {
-    ver = ["8", "9", "10"]
-  }
-  args = {
-    version = ver
-  }
-  tags = ["${IMAGE_URI}/gcc-version-${ver}:${TAG}"]
-}
-
-target "gcc-compilers-base-22-04" {
-  name = "gcc-compilers-${replace(ver, ".", "-")}"
-  inherits = ["common"]
-  context = "ubuntu-22.04_gcc-version"
-  dockerfile = "Dockerfile"
-  matrix = {
-    ver = ["11", "12"]
-  }
-  args = {
-    version = ver
-  }
-  tags = ["${IMAGE_URI}/gcc-version-${ver}:${TAG}"]
-}
-
-target "clang-compilers-base-20-04" {
-  name = "clang-compilers-${replace(ver, ".", "-")}"
+target "gcn-gpu" {
   inherits = ["common"]
-  context = "ubuntu-20.04_clang-version"
+  platform = "linux/amd64" # Only build for x86.
   dockerfile = "Dockerfile"
-  matrix = {
-    ver = ["7", "8", "9", "10", "11", "12"]
-  }
-  args = {
-    version = ver
-  }
-  tags = ["${IMAGE_URI}/clang-version-${ver}:${TAG}"]
+  context = "gcn-gpu"
+  tags = ["${IMAGE_URI}/gcn-gpu:${TAG}"]
 }
 
-target "clang-compilers-base-22-04" {
-  name = "clang-compilers-${replace(ver, ".", "-")}"
+target "gpu-fs" {
```

Comment:
What is this docker image used for?

Reply:
I believe 22.04 is still used, but @abmerop can confirm

Reply:
@mattsinc : I'm pretty sure Jason is asking about the "gpu-fs" docker image. The 22-04 image still exists.

Reply:
Actually, i can't even build this image. I get `Unable to locate package rocm-dev`. @mattsinc : you last updated the "gpu-fs/Dockerfile", I think you may have broken it in the process?

Reply:
Nevermnind, it started building for some reason... I don't know why...

Reply:
This is used to build GPU applications. It contains the GPU compiler and libraries. You can think of it like a "cross compiler" docker.

Reply:
As long as you still use it, that's good enough for me.